controllers/v1/userProjects.js

```diff
@@ -1199,7 +1199,10 @@ module.exports = class UserProjects extends Abstract {
 		return new Promise(async (resolve, reject) => {
 			try {
 				// ReIssue certificate of given project : projectId is passed as param
-				let projectDetails = await userProjectsHelper.certificateReIssue(req.params._id)
+				let projectDetails = await userProjectsHelper.certificateReIssue(
```

Comment:
@Prajwal17Tunerlabs if we are not storing the user Profile , it s better to call user service directly

Reply:
user service changes for certificateReissue api reverted.